### ISSUE TITLE
BottleneckCSP does not follow the cfg file of CSPDarknet53

### ISSUE THREAD

**Issue by vladhondru25** [ultralytics/yolov5#1791]
After carefully looking at the cfg file for the CSPDarknet53 provided in the [repository](https://github.com/WongKinYiu/CrossStagePartialNetworks/blob/master/cfg/csdarknet53.cfg) for the CSPNet, I noticed a discrepancy. 

The cfg file states the following (lines 49-55):
`[convolutional]
batch_normalize=1
filters=64
size=1
stride=1
pad=1
activation=leaky`
https://github.com/WongKinYiu/CrossStagePartialNetworks/blob/8786894eec2bc8e7d78572d5a770f091baa6f4fb/cfg/csdarknet53.cfg#L49-L55

This is corresponding to "self.cv2" in the BottleneckCSP:
https://github.com/ultralytics/yolov5/blob/14b0abe2d0b0cc0f2a465fb534f7054fb7207caa/models/common.py#L55-L71

Shouldn't "self.cv2" be followed by a BatchNormalisation and an activation function (LeakyRelu) as per cfg file?

The same situation applies to "self.cv3".

Thank you very much!

**Comment by vladhondru25** [comment]
Actually, it is correct. instead of two separate layers and the concatenate, the code concatenates first and then applies the BN and activation, which should be equivalent.

**Comment by glenn-jocher** [comment]
@vladhondru25 yes, your second observation is correct.
